game/server/game/models.py

```diff
@@ -28,6 +28,7 @@ class GameJs(BaseModel):
     threat_zones: ThreatZoneContainerJs
     navmeshes: NavMeshesJs
     map_center: LeafletPoint | None
+    unculled_zones: list[UnculledZoneJs]
```

Comment:
Back-end changes for changing the API, required for `gameLoaded` in `unculledZonesSlice`